auto/lastfm.q

```diff
@@ -0,0 +1,17 @@
+\d .lfm
+
+tm:{
+  .lg.o"Running last.fm charts timer";
+  if[not .lfm.valid[];:.lg.w"last.fm not enabled, chart timer aborted"];                        / exit if functionality disabled
+  if[0=count .lfm.users;:.lg.w"No last.fm usernames provuded, chart timer aborted"];            / exit if no cached usernames
+  if[0=count .lfm.o.charts;:.lg.w"No charts currently specified in .lfm.o.charts"];
+  c:chart[];                                                                                    / get charts
+  .lg.o"Sending last.fm charts to slack";
+  .slack.msg[.slack.hooks .lfm.channel]c;                                                       / post chart
```

Comment:
I want this to be published by `lastfmbot`,, is this close to what I need?
```
.slack.postase[c;x`channel_id;"lastfmbot";":lastfm:"];
```
I'm just not sure what `channel_id` is and I have the channel name stored in `.lfm.channel`.

Reply:
`.slack.chanlist` is a dict mapping channel names to IDs, keys are channel names as strings so it'd be something like

```q
.slack.chanlist"qradio"
```

to get the correct ID